scripts/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "../.eslintrc.json"

```

Comment:
Is this file needed?

Reply:
@uniqueiniquity this file can be removed.